pkg/scheduler/framework/interface.go

```diff
@@ -148,7 +150,7 @@ type Status struct {
 // Code returns code of the Status.
 func (s *Status) Code() Code {
 	if s == nil {
-		return Success
+		return None
```

Comment:
when I was writing the test, I noticed that it is very strange that it is Success when it is nil.
Is this by design of the Scheduler?
Shouldn't you add a status like `None` exactly, where Status is nothing?

Reply:
It's as design. You can refer to https://github.com/kubernetes/kubernetes/blob/b36c927d957e585ce54d415be4d7e4621e189d85/pkg/scheduler/framework/interface.go#L62-L64.

Reply:
thank you for your comment.
I had missed that reference.ea